scss/base/_variables.scss

```diff
@@ -132,10 +132,6 @@ $zindex-ad-top-bar:                       12600 !default;
 $zindex-popover:                          12600 !default;
 $SUPER_Z_INDEX:                           20000 !default; //in core, here for bird's eye view tracking
 $zindex-fullscreen:                  2147483647 !default; //in screenfull, here for bird's eye view tracking
-$zindex-state-screen:                      1000 !default;
-$zindex-vr-controls:                       1001 !default;
-$zindex-watermark:                         1001 !default;
-$zindex-up-next-panel:                     1002 !default;
```

Comment:
Just to confirm, these are the values that had been added for VR360 changes so this would take us back to the original state, correct?

Reply:
The git blame show Kseniia is the last to touch it.  
be8c8c626 (Kseniia Sokolova  2017-10-13 16:27:39 +0300 135) $zindex-state-screen:                      1000 !default;
be8c8c626 (Kseniia Sokolova  2017-10-13 16:27:39 +0300 136) $zindex-vr-controls:                       1001 !default;
ff4b81c55 (Kseniia Sokolova  2017-12-04 17:39:47 +0300 137) $zindex-watermark:                         1001 !default;
7c9ace1b5 (Kseniia Sokolova  2017-10-16 15:44:23 +0300 138) $zindex-up-next-panel:                     1002 !default;

Reply:
And before that they were renamed to their current form. So I'm confident they were added for vr.